src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+//last modified by sebrianne ferguson on 2/23/2019
```

Comment:
This is not necessary. github will track these changes (who, what, when). Please remove it.

Reply:
done.